site2/docs/client-libraries-cpp.md

```diff
@@ -4,298 +4,103 @@ title: Pulsar C++ client
-| [client-debuginfo](@pulsar:dist_rpm:client-debuginfo@) | [asc](@pulsar:dist_rpm:client-debuginfo@.asc),  [sha512](@pulsar:dist_rpm:client-debuginfo@.sha512) |
-| [client-devel](@pulsar:dist_rpm:client-devel@) | [asc](@pulsar:dist_rpm:client-devel@.asc),  [sha512](@pulsar:dist_rpm:client-devel@.sha512) |
+1. Download any one of the RPM packages:
 
-2. Install the package using the following command.
+<Tabs>
+<TabItem value="client">
 
 ```bash
-rpm -ivh apache-pulsar-client*.rpm
+wget @pulsar:dist_rpm:client@
 ```
 
-After you install RPM successfully, Pulsar libraries are in the `/usr/lib` directory, for example：
+</TabItem>
+<TabItem value="client-debuginfo">
 
 ```bash
-lrwxrwxrwx 1 root root 18 Dec 30 22:21 libpulsar.so -> libpulsar.so.2.9.1
-lrwxrwxrwx 1 root root 23 Dec 30 22:21 libpulsarnossl.so -> libpulsarnossl.so.2.9.1
+wget @pulsar:dist_rpm:client-debuginfo@
 ```
 
-:::note
-
-If you get the error that `libpulsar.so: cannot open shared object file: No such file or directory` when starting Pulsar client, you may need to run `ldconfig` first.
-
-:::
-
-2. Install the GCC and g++ using the following command, otherwise errors would occur in installing Node.js.
+</TabItem>
+<TabItem value="client-devel">
 
 ```bash
-sudo yum -y install gcc automake autoconf libtool make
-sudo yum -y install gcc-c++
+wget @pulsar:dist_rpm:client-devel@
 ```
 
-### Install Debian
-
-1. Download a Debian package from the links in the table.
-
-| Link | Crypto files |
-|------|--------------|
-| [client](@pulsar:deb:client@) | [asc](@pulsar:dist_deb:client@.asc), [sha512](@pulsar:dist_deb:client@.sha512) |
-| [client-devel](@pulsar:deb:client-devel@) | [asc](@pulsar:dist_deb:client-devel@.asc),  [sha512](@pulsar:dist_deb:client-devel@.sha512) |
+</TabItem>
+</Tabs>
 
-2. Install the package using the following command.
+2. Install the package using the following command:
 
 ```bash
-apt install ./apache-pulsar-client*.deb
-```
-
-After you install DEB successfully, Pulsar libraries are in the `/usr/lib` directory.
-
-### Build
-
-If you want to build RPM and Debian packages from the latest master, follow the instructions below. You must run all the instructions at the root directory of your cloned Pulsar repository.
-
-There are recipes that build RPM and Debian packages containing a
-statically linked `libpulsar.so` / `libpulsarnossl.so` / `libpulsar.a` / `libpulsarwithdeps.a` with all required dependencies.
-
-To build the C++ library packages, you need to build the Java packages first.
-
-```shell
-mvn install -DskipTests
-```
-
-#### RPM
-
-To build the RPM inside a Docker container, use the command below. The RPMs are in the `pulsar-client-cpp/pkg/rpm/RPMS/x86_64/` path.
-
-```shell
-pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh
-```
-
-| Package name | Content |
-|-----|-----|
-| pulsar-client | Shared library `libpulsar.so` and `libpulsarnossl.so` |
```

Comment:
Why remove this table? This table explains the differences between the individual packages.

Reply:
See https://github.com/apache/pulsar/pull/17580#discussion_r967677766.

Reply:
OK. I get your point. Will push a commit later. FYI - It's not a regression. Previously we don't talk about it in the installation section also.

Reply:
I don't quite understand. For the user, he can choose these three packages to use, but we didn't introduce the difference between these three packages.

<img width="1291" alt="image" src="https://user-images.githubusercontent.com/33416836/190299259-a328b0ce-9442-4c4e-8b45-80562cd9bc65.png">

Reply:
I have the same question.

Reply:
But just as tison said, it's not a regression, we can explain it in another PR.